1/p와-y개수-세기.js

```diff
@@ -0,0 +1,22 @@
+// p와 y개수 세기
+
+function solution(s) {
+  let answer = true;
+  let pNum = 0; // p 개수 저장 변수
+  let yNum = 0; // y 개수 저장 변수
+  for (let i of s) {
```

Comment:
```suggestion
  let yCnt = 0; // y 개수 저장 변수
  
  for (let i of s) {
```
변수 선언 후 구문은 줄 바꿈 후 작성합니다.